refactor(curriculum): update RPG project to accommodate new intro pyramid project (#54435)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa27c40ca6f04ab8be5fac.md

---
id: 62aa27c40ca6f04ab8be5fac
title: Step 166
challengeType: 0
dashedName: step-166
---

# --description--

In the previous project, you learned how to work with `for` loops like this:

```js
for (let i = 0; i < 5; i++) {
  // code to run
}
```

> `for` loops are declared with three expressions separated by semicolons. `for (a; b; c)`, where `a` is the initialization expression, `b` is the condition, and `c` is the final expression.

Reviewer: ```suggestion
`for` loops are declared with three expressions separated by semicolons: `for (a; b; c)`, where `a` is the initialization expression, `b` is the condition, and `c` is the final expression.
```

> In this step, create a `for` loop where `i` is initialized to `0`, the loop runs while `i` is less than `10`, and `i` is incremented by `1` after each loop. Remember to use the increment operator `++` to increase the value of `i` by `1`.

Reviewer: ```suggestion
In this step, create a `for` loop where `i` is initialized to `0`, the loop runs as long as `i` is less than `10`, and `i` is incremented by `1` after each iteration using the increment operator `++`.
```
